### DISCUSSION TITLE
In case of using calibration plugin, xDrip statistics work with raw data

### DISCUSSION THREAD

**Discussion by konstantinov** [NightscoutFoundation/xDrip#3797] [Bug reports]
Hey! I'm using xDrip with calibration plugin `xDrip original - classical algorithm` and faced issue: statistics graph build based on raw data, not on a calibrated. Is it a source code issue? Maybe I missed something in settings?

![image|10](https://github.com/user-attachments/assets/63e1701f-0cc5-4f3a-9a3c-e1a17b71229e)
![image](https://github.com/user-attachments/assets/92b56e28-b801-4257-ad28-e8679f78744d)
![image](https://github.com/user-attachments/assets/acfea867-b4cc-4ec7-bf27-3377e19f8032)

**Comment by Navid200** [comment]
I don't think using the plugin is recommended.  I believe it is a last resort.
If you are using a sensor that does not offer calibration, you can contact the manufacturer to find out why they don't believe there is a need for it.

xDrip should not get into a head-to-head contradictory approach with manufacturers.
xDrip offers solutions when there is a missing necessity.  
Calibration is not a requirement as much as it used to be 10 years ago.

Can you tell us what sensor you use?

**Reply by konstantinov** [reply]
@Navid200 thanks for quick response. I'm using Sibionics sensor with Juggluco app. It was working fine until I've got a difference in laboratory value and sibionics sensor. So I thought calibration may help with this.
